README.md

```diff
@@ -1933,9 +1933,9 @@ void UGDGameplayAbility::OnAvatarSet(const FGameplayAbilityActorInfo * ActorInfo
 {
 	Super::OnAvatarSet(ActorInfo, Spec);
 
-	if (ActivateAbilityOnGranted)
+	if (bActivateAbilityOnGranted)
```

Comment:
If we change it here, we should change it in the sample project code too.

Reply:
I would like to note that the next line, i.e. the line of the unused local variable was changed/fixed in the first revert commit, like you did in the 5.1 branch. Let me know if that's okay 😄

Reply:
Ah fair enough, that's also in the sample code which I missed. I fixed that in the new PR: https://github.com/tranek/GASDocumentation/pull/95